README.md

```diff
@@ -42,10 +42,10 @@ Promote awareness about your CFP in the community:
 
 # Getting a design approved
 
-For a CFP to be considered viable, a Cilium committer needs to aprove it.
-After the approval, the design can be merged. A merged design proposal 
-means the proposal is viable to be executed on, but not that there is a
-100% chance it will be accepted.
+For a CFP to be considered implementable, a Cilium committer needs to approve it.
+After their approval, the design can be merged. A merged design proposal 
+means the proposal is viable to be implemented, but not that there is a
+100% chance the code will be accepted.
```

Comment:
```suggestion
100% chance the code will be accepted. Dormant and declined CFPs are
also an important part of documentation for the project. CFPs with theses
statuses can also be merged with approval from one committer.
```

Reply:
^ Re: last suggestion, first line already mentions committer approval - no need to mention it twice.

Reply:
ok, slightly changed around the wording to reflect that